docs/questions/sharing/visualizations/sankey.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Sankey charts
+---
+# Sankey charts
+
+Sankey charts show how data flows through multi-dimensional steps. They're useful for showing which elements, called nodes, contribute to the overall flow.
+
+![Left-aligned sankey chart](../../images/sankey-left-aligned.png)
+
+## Sankey data shape
+
+To create a Sankey chart, you'll need at least three columns in your results:
+
+- **Source column**: specifies a node in the sankey flow.
+- **Target column**: specifies a receiving node.
+- **Count column**: value that determines the thickness of the target node.
+
+Here's the data shape used to create the chart above.
```

Comment:
This is great as an example but tbh I'm really confused how to build a QB question that would give a result like this